sdk/communication/Azure.Communication.Rooms/CHANGELOG.md

```diff
@@ -1,14 +1,7 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
```

Comment:
correct me if i'm wrong @mikehang-msft, you might have some knowledge here too - 
the release notes here don't need to be removed (the contribution guide might be clearer on how to maintain the release notes/versions)

Reply:
Thanks @allchiang-msft, looking at the history, those are added automatically by a bot.

Reply:
Thanks!